DOCUMENTATION.md

```diff
@@ -1,36 +1,52 @@
-## Table of Contents
+# Table of Contents
 * [Description](#description)
 * [MQTTClient](#mqttclient)
-* [connect](#connect)
-* [close](#close)
-* [set_uplink_callback](#set_uplink_callback)
-  * [uplink_callback](#uplink_callback)
-* [set_connect_callback](#set_connect_callback)
-  * [connect_callback](#connect_callback)
-* [set_downlink_callback](#set_downlink_callback)
-  * [downlink_callback](#downlink_callback)
-* [set_close_callback](#set_close_callback)
-  * [close_callback](#close_callback)
-* [send](#send)
-* [Errors](#errors)
-* [UplinkMessage](#uplinkmessage)
+  * [connect](#connect)
+  * [close](#close)
+  * [set_uplink_callback](#set_uplink_callback)
+    * [uplink_callback](#uplink_callback)
+  * [set_connect_callback](#set_connect_callback)
+    * [connect_callback](#connect_callback)
+  * [set_downlink_callback](#set_downlink_callback)
+    * [downlink_callback](#downlink_callback)
+  * [set_close_callback](#set_close_callback)
+    * [close_callback](#close_callback)
+  * [send](#send)
+  * [UplinkMessage](#uplinkmessage)
+* [ApplicationClient](#applicationclient)
+  * [get](#get)
+  * [set_payload_format](#set_payload_format)
+  * [set_custom_payload_functions](#set_custom_payload_functions)
+  * [unregister](#unregister)
+  * [device](#device)
+  * [devices](#devices)
+  * [update_device](#update_device)
+  * [delete_device](#delete_device)
+  * [DeviceObject](#deviceobject)
+  * [Application](#application)
+* [HandlerClient](#handlerclient)
+  * [data](#data)
+  * [application](#application)
+* [Troubleshooting](#troubleshooting)
 
-## Description
 
-This package provides you an easy way to connect to The Things Network via MQTT.
+## Description
+This package provides you an easy way to exchange traffic with The Things Network via MQTT and manage your applications.
 
-### MQTTClient
 
+## MQTTClient
 The class constructor can be called following this scheme:
 ```python
-MQTTClient(app_id, app_access_key, [mqtt_address], [discovery_address])
+MQTTClient(app_id, app_access_key, **kwargs)
```

Comment:
I'm not a fan of not specifying named parameters in the signature of the function. We don't have a lot of named parameters, stick to specifying them in the signature. This is how the [Saltstack documentation](https://docs.saltstack.com/en/latest/ref/states/all/salt.states.k8s.html) is done for example, it's pretty nice and it's easier to read, and also it indicates the default values.
Remove `kwargs` and set named parameters with default values instead.